### DISCUSSION TITLE
Test results of multiple cards

### DISCUSSION THREAD

**Discussion by Lost-Entrepreneur439** [vanilla-wiiu/vanilla#23] [Hardware Results]
I've decided to test with the Wi-Fi card on my current laptop, an Intel Centrino Advanced-N 6205, and despite being over 10 years old, I am happy to report that this card works flawlessly with Vanilla, paired first try, and works with a good framerate. Touch works, display works, input works, microphone is untested as my laptop does not have a microphone.
If it's relevant, my system is running Debian Testing (Trixie).

**Comment by VoxelTek** [comment]
Cool, will add this to the compatibility table. Thanks for your input! 👍🏻

**Comment by Lost-Entrepreneur439** [comment]
I have more cards to test asides from the Centrino, I will add more edits to this issue as I test more cards.
### Broadcom BCM43241
**Not working.** Pairs fine, but when attempting to connect, loops "INFO WAITING FOR CONNECTION" over and over and over again.

**Comment by Lost-Entrepreneur439** [comment]
### Intel Wireless AC-9560
**Works perfectly.** Like all other Intel cards, the 9560 has absolutely no trouble with Vanilla. The 9560 is used in the Lenovo 500e Chromebook 2nd gen, and probably most Gemini Lake systems.

**Reply by TCAtrevor** [reply]
This chipset is not working for me, tested in a live Debian environment.

**Reply by VoxelTek** [reply]
@TCAtrevor given that you're having issues no one else is, I'm guessing it might not be your WiFi card. Maybe check the log output, and/or start an issue?

**Reply by TCAtrevor** [reply]
Log outputs did look weird, I attached them on my other tests. I'll look into making an issue.

**Comment by Pepeshki** [comment]
This card is also used on the ThinkPad T430
